### PR TITLE
BENCH 106: Create Docker Image for Answer King C# app

### PR DESCRIPTION
**Pull request by mbaltrimas**
adds an action that builds a docker image every time a push is made to `main` or `develop`

currently doesn't push the image anywhere as its a separate ticket

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
```
Comment (milkshakeuk) [inline dockerfile:1]: this docker file will re-build the binaries which might have already been built as part of other actions... not necessarily an issue but something that could be optimised later.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Workflow for building a docker image
```
Comment (milkshakeuk) [inline .github/workflows/build.yml]: we should probably rename this file to `docker-build.yml` for now.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+name: Build docker image
+on:
+  push:
+    branches:
```
Comment (milkshakeuk) [inline .github/workflows/build.yml]: do we want to do this on every push to `main` and `develop` ? I guess it's fine for now but something to think about and maybe discuss.

**Review by milkshakeuk — CHANGES_REQUESTED** [review]
Just the one change for now 😊

**Review by milkshakeuk — APPROVED** [review]
Looks good!

**Review by harrystead2 — APPROVED** [review]

**Pull request merged**